packages/server/tests/acceptance/htsPrecompile.spec.ts

```diff
@@ -586,6 +586,65 @@ describe('HTS Precompile Acceptance Tests', async function () {
     });
   });
 
+  // Depends on https://github.com/hashgraph/hedera-services/pull/3756
+  xdescribe('HTS Precompile Token Expiry Info Tests', async function() {
+    it('should be able to get fungible token expiry info', async function() {
+      const getTokenExpiryInfoTx = (await baseHTSContract.getTokenExpiryInfoPublic(HTSTokenContractAddress));
+      const responseCode = (await getTokenExpiryInfoTx.wait()).events.filter(e => e.event === 'ResponseCode')[0].args.responseCode;
+      const tokenExpiryInfo = (await getTokenExpiryInfoTx.wait()).events.filter(e => e.event === 'TokenExpiryInfo')[0].args.expiryInfo;
+
+      expect(responseCode).to.equal(TX_SUCCESS_CODE);
+      expect(tokenExpiryInfo.autoRenewPeriod).to.equal(7776000);
+      expect(tokenExpiryInfo.autoRenewAccount).to.equal(HTSTokenContractAddress);
+    });
+
+    it('should be able to get non fungible token expiry info', async function() {
+      const getTokenExpiryInfoTx = (await baseHTSContract.getTokenExpiryInfoPublic(NftHTSTokenContractAddress));
+      const responseCode = (await getTokenExpiryInfoTx.wait()).events.filter(e => e.event === 'ResponseCode')[0].args.responseCode;
+      const tokenExpiryInfo = (await getTokenExpiryInfoTx.wait()).events.filter(e => e.event === 'TokenExpiryInfo')[0].args.expiryInfo;
+
+      expect(responseCode).to.equal(TX_SUCCESS_CODE);
+      expect(tokenExpiryInfo.autoRenewPeriod).to.equal(7776000);
+      expect(tokenExpiryInfo.autoRenewAccount).to.equal(NftHTSTokenContractAddress);
+    });
+
+    it('should be able to update fungible token expiry info', async function() {
```

Comment:
Improvement suggestion.
Change these 4 tests into 2 tests

Test 1 ( combines your current test 1 and 3)
- confirm `getTokenExpiryInfoPublic()` for FT
- update expiry info using `updateTokenExpiryInfoPublic()`
- confirm updated info


Test 2 ( combines your current test 2 and 4)
- confirm `getTokenExpiryInfoPublic()` for NFT
- update expiry info using `updateTokenExpiryInfoPublic()`
- confirm updated info

Reply:
Done. Updated the flow.